src/components/DistanceRequest.js

```diff
@@ -312,7 +327,7 @@ function DistanceRequest({transactionID, report, transaction, mapboxAccessToken,
                     <View style={[styles.flex1, safeAreaPaddingBottomStyle]}>
                         <HeaderWithBackButton
                             title={translate('common.distance')}
-                            onBackButonBackButtonPress={navigateBack}
+                            onBackButtonPress={navigateBack}
```

Comment:
I noticed a typo here, should we handle it in a separate PR? Or it's fine to fix it here? cc @hayata-suenaga

Reply:
Seems you already fixed it

Reply:
Yes, but it's still broken on main until this PR is merged. I'm just wondering if it's fine to fix it here or we need to fix it asap on a different branch